ctapipe/io/astropy_helpers.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+import tables
+from astropy.table import Table
+from astropy.units import Unit
+
+__all__ = ["h5_table_to_astropy"]
+
+
+def h5_table_to_astropy(h5file, path) -> Table:
```

Comment:
Maybe use `QTable`? In the end the point of this function is to provide access to the units. Maybe this could also be an argument. `table_class=QTable`

Reply:
The difference is that accessing columns or rows or single elements yields simple numbers for `Table` but quantities for `QTable`:

Reply:
I'm not sure Qtables are very standard or used anywhere, units still work in normal Tables.

```
table['energy'].quantity  # → returns a Quantity array for that column
table['energy'].to("GeV") # returns a Column in GeV
```

Reply:
The difference is mainly for iterating over the rows:
```
for row in table:
    print(row['energy')  # float for Table, Quantity for QTable
```

Reply:
Ok, will change to QTable